index.test-d.ts

```diff
@@ -0,0 +1,29 @@
+import {expectType} from 'tsd-check';
+import pSome, {AggregateError} from '.';
+import PCancelable from 'p-cancelable';
```

Comment:
Should we maybe instead re-export `PCancelable` as a `CancelablePromise` type in the type definition, so users don't need to get the type from a sub-dependency?

Reply:
True.